src/components/LocationPage/NewLocationPageHeader.tsx

```diff
@@ -0,0 +1,200 @@
+        {props.projections.stateCode}
+      </a>
+    </>
+  ) : (
+    <strong>{props.projections.stateName}</strong>
+  );
+
+  return <span>{displayName}</span>;
+};
+
+const noop = () => {};
+
+const NewLocationPageHeader = (props: {
+  projections: Projections;
+  condensed?: Boolean;
+  stats: { [key: string]: number | null };
+  onRtRangeClick?: () => void;
+  onTestPositiveClick?: () => void;
+  onIcuUtilizationClick?: () => void;
+  onContactTracingClick?: () => void;
+  onHeaderShareClick: () => void;
+  onHeaderSignupClick: () => void;
+  isMobile?: Boolean;
+}) => {
+  const hasStats = !!Object.values(props.stats).filter(
+    (val: number | null) => !isNull(val),
+  ).length;
+
+  const headerTopMargin = !hasStats ? -270 : -330;
+  const headerBottomMargin = !hasStats ? 100 : 0;
+
+  const locationName =
+    props.projections.countyName || props.projections.stateName;
+
+  const alarmLevel = props.projections.getAlarmLevel();
+
+  const levelInfo = LOCATION_SUMMARY_LEVELS[alarmLevel];
+
+  const fillColor =
+    alarmLevel !== Level.UNKNOWN ? levelInfo.color : COLOR_MAP.GRAY.LIGHT;
+
+  const { isEmbed } = useEmbed();
+
+  const verified = STATES_WITH_DATA_OVERRIDES.includes(
+    props.projections.stateName,
+  );
+
+  const isVerifiedState = !props.projections.isCounty && verified;
+
+  const lastUpdatedDate: Date | null = useModelLastUpdatedDate() || new Date();
+  const lastUpdatedDateString =
+    lastUpdatedDate !== null ? formatDate(lastUpdatedDate) : '';
+
+  const riskLevelGraphicProps = {
+    alarmLevel,
+    levelName: levelInfo.name,
+    projections: props.projections,
+  };
+
+  return (
+    <Fragment>
+      <ColoredHeaderBanner bgcolor={fillColor} hasStats={hasStats} />
+      <HeaderContainer
+        condensed={props.condensed}
+        headerTopMargin={headerTopMargin}
+        headerBottomMargin={headerBottomMargin}
+      >
+        <HeaderSection>
+          <LocationCopyWrapper>
+            <HeaderTitle isEmbed={isEmbed}>
+              <LocationPageHeading projections={props.projections} />
+            </HeaderTitle>
+            <RiskLevelGraphicMobile>
+              <RiskLevelGraphic {...riskLevelGraphicProps} />
+            </RiskLevelGraphicMobile>
+            <HeaderSubtitle>{levelInfo.detail(locationName)}</HeaderSubtitle>
+          </LocationCopyWrapper>
+          <RiskLevelGraphicDesktop>
+            <RiskLevelGraphic {...riskLevelGraphicProps} />
+          </RiskLevelGraphicDesktop>
+        </HeaderSection>
+        <SummaryStats
+          stats={props.stats}
+          onRtRangeClick={props.onRtRangeClick}
+          onTestPositiveClick={props.onTestPositiveClick}
+          onIcuUtilizationClick={props.onIcuUtilizationClick}
+          onContactTracingClick={props.onContactTracingClick}
+          isMobile={props.isMobile}
+        />
+        <HeaderSection>
+          <HeaderSubCopyWrapper isVerifiedState={isVerifiedState}>
+            {props.projections.isCounty && !isEmbed && (
```

Comment:
Seems like all of this embed-specific stuff could be removed once this is merged with embeds 2.0?  (not urgent)

Reply:
Yeah, same as below- I'm gonna rip out the old embed+old header code once the new ones are deployed